lib/PuppeteerSharp/BrowserContext.cs

```diff
@@ -122,21 +101,22 @@ public Task CloseAsync()
         /// ]]>
         /// </example>
         /// <seealso href="https://developer.mozilla.org/en-US/docs/Glossary/Origin"/>
-        public async Task OverridePermissionsAsync(string origin, IEnumerable<OverridePermission> permissions)
+        public Task OverridePermissionsAsync(string origin, IEnumerable<OverridePermission> permissions)
         {
-            var permissionList = permissions.Select(permission =>
+            var invalidPermission = permissions
```

Comment:
can `invalidPermission` actually have a value?

Reply:
Yes here. https://github.com/kblok/puppeteer-sharp/pull/661/files#diff-89c4d2b1dab8fadf851afd2b813aff2dR42

I didn't ping you yet because I'm trying to find better ways to code that part but "it's working"

Reply:
I don't think we should care for that case.
you can always come up with weird cases, I think having it use an enum is enough